packages/paste-core/components/side-panel/src/SidePanel.tsx

```diff
@@ -49,6 +49,7 @@ const StyledSidePanelWrapper = React.forwardRef<HTMLDivElement, BoxProps>((props
     width={["100%", "size40", "size40"]}
     height={props.height}
     boxSizing="content-box"
+    backgroundColor="colorBackgroundWeaker"
   />
 ));
 
```

Comment:
question: Do the borders still need to be swapped out for a shadow? https://docs.google.com/document/d/1WzOL9RyUL5eX41FlMog7gVDGB0RsZyQXcJI6GsADKdk/edit?tab=t.0#heading=h.8g82d22xdpz Or did y'all talk about keeping the borders?